bot/main.py

```diff
@@ -27,6 +28,16 @@
     logging.basicConfig(level=LOGGING_LEVEL)
 
 
+spam_log_file_clear_interval = 3
+spam_log_file = str(
```

Comment:
Some nice new syntax using the `pathlib` is e.g.:

```python
spam_log_file = pathlib.Path(os.getcwd()) / "bot" / "cogs" / "spam" / "spam_log.txt"
```

Reply:
Spam file creation moved from main to the cog and cleanup in commit d16a01045b4ac4dd66224f6f801cdc23297a2c64 👍